backend/prisma/migrations/20231105081703_change_model_name_to_singular/migration.sql

```diff
@@ -0,0 +1,59 @@
+/*
+  Warnings:
+
+  - You are about to drop the `publications` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `publications_traveler` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `travelers` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "publications_traveler" DROP CONSTRAINT "publications_traveler_publication_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "publications_traveler" DROP CONSTRAINT "publications_traveler_traveler_id_fkey";
+
+-- DropTable
+DROP TABLE "publications";
+
+-- DropTable
+DROP TABLE "publications_traveler";
+
+-- DropTable
+DROP TABLE "travelers";
+
+-- CreateTable
+CREATE TABLE "traveler" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "traveler_name" VARCHAR(1000) NOT NULL,
+    "traveler_type" "traveler_type" NOT NULL DEFAULT 'AUTHOR',
+
+    CONSTRAINT "traveler_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "publication" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "title" VARCHAR(2000) NOT NULL,
+    "summary" VARCHAR(2000) NOT NULL,
+    "can_read" BOOLEAN NOT NULL,
+
+    CONSTRAINT "publication_pkey" PRIMARY KEY ("id")
```

Comment:
Is there a reason why we aren't just naming id to publication_pkey and setting it as a primary key directly?